log/log.go

```diff
@@ -1,71 +1,118 @@
-func Error(args ...interface{})                             { std.Error(args...) }
-func Errorf(format string, args ...interface{})             { std.Errorf(format, args...) }
-func Errorln(args ...interface{})                           { std.Errorln(args...) }
-func Info(args ...interface{})                              { std.Info(args...) }
-func Infof(format string, args ...interface{})              { std.Infof(format, args...) }
-func Infoln(args ...interface{})                            { std.Infoln(args...) }
-func Print(args ...interface{})                             { std.Print(args...) }
-func Printf(format string, args ...interface{})             { std.Printf(format, args...) }
-func Println(args ...interface{})                           { std.Println(args...) }
-func Fatal(args ...interface{})                             { std.Fatal(args...) }
-func Fatalf(format string, args ...interface{})             { std.Fatalf(format, args...) }
-func Fatalln(args ...interface{})                           { std.Fatalln(args...) }
-func Warn(args ...interface{})                              { std.Warn(args...) }
-func Warnf(format string, args ...interface{})              { std.Warnf(format, args...) }
-func Warnln(args ...interface{})                            { std.Warnln(args...) }
-func WithField(key string, value interface{}) *logrus.Entry { return std.WithField(key, value) }
-func WithFields(fields logrus.Fields) *logrus.Entry         { return std.WithFields(fields) }
+func Print(args ...interface{})                 { Info(args...) }
+func Printf(format string, args ...interface{}) { Infof(format, args...) }
+func Println(args ...interface{})               { Infoln(args...) }
+
+func Debug(args ...interface{}) { std.Debug(fmt.Sprint(args...)) }
+func Error(args ...interface{}) { std.Error(fmt.Sprint(args...)) }
+func Info(args ...interface{})  { std.Info(fmt.Sprint(args...)) }
+func Warn(args ...interface{})  { std.Warn(fmt.Sprint(args...)) }
+func Fatal(args ...interface{}) { std.Fatal(fmt.Sprint(args...)) }
+
+func Debugf(format string, args ...interface{}) { std.Debug(fmt.Sprintf(format, args...)) }
+func Errorf(format string, args ...interface{}) { std.Error(fmt.Sprintf(format, args...)) }
+func Infof(format string, args ...interface{})  { std.Info(fmt.Sprintf(format, args...)) }
+func Warnf(format string, args ...interface{})  { std.Warn(fmt.Sprintf(format, args...)) }
+func Fatalf(format string, args ...interface{}) { std.Fatal(fmt.Sprintf(format, args...)) }
+
+func Debugln(args ...interface{}) { std.Debug(fmt.Sprintln(args...)) }
+func Errorln(args ...interface{}) { std.Error(fmt.Sprintln(args...)) }
+func Infoln(args ...interface{})  { std.Info(fmt.Sprintln(args...)) }
+func Fatalln(args ...interface{}) { std.Fatal(fmt.Sprintln(args...)) }
+func Warnln(args ...interface{})  { std.Warn(fmt.Sprintln(args...)) }
+
+func Debugw(msg string, fields ...zapcore.Field) { std.Debug(msg, fields...) }
```

Comment:
What `*w` stands for? "With"? :)

Reply:
just following zap conventions, but yes, probably: https://godoc.org/go.uber.org/zap#SugaredLogger.Infow